src/aggregate/aggregate_request.c

```diff
@@ -1285,7 +1291,11 @@ static ResultProcessor *getScorerRP(AREQ *req) {
   IndexSpec_GetStats(req->sctx->spec, &scargs.indexStats);
   scargs.qdata = req->ast.udata;
   scargs.qdatalen = req->ast.udatalen;
-  ResultProcessor *rp = RPScorer_New(fns, &scargs);
+  const RLookupKey *scoreKey = NULL;
+  if (req->reqflags & QEXEC_F_SEND_SCORES_AS_FIELD) {
+    scoreKey = RLookup_GetKey(rl, UNDERSCORE_SCORE, RLOOKUP_M_WRITE, RLOOKUP_F_NOFLAGS);
```

Comment:
Where was `UNDERSCORE_SCORE` defined? was it in use before?

Reply:
It is defined along with `UNDERSCORE_KEY`. Not sure if it is being used anywhere today, I think it it’s related to the default score field name with `FT.ADD`.